language/de.json

```diff
@@ -1,292 +1,321 @@
 {
-  "semantics": [
+  "semantics":[
     {
-      "fields": [
+      "fields":[
         {
-          "field": {
-            "fields": [
+          "field":{
+            "fields":[
               {
-                "field": {
-                  "fields": [
-                    {},
-                    {},
-                    {},
-                    {},
-                    {},
+                "field":{
+                  "fields":[
                     {
-                      "label": "Kommentare",
-                      "description": "Die Kommentare werden angezeigt, wenn der Nutzer die vorgeschlagenen Antworten für alle Schaubilder anzeigt."
+                      
                     },
                     {
-                      "label": "Kommentare immer anzeigen"
+                      
                     },
                     {
-                      "label": "Undurchlässigkeit des Hintergrunds"
+                      
                     },
                     {
-                      "label": "Als Button anzeigen"
+                      
                     },
                     {
-                      "label": "Titel"
+                      
                     },
                     {
-                      "label": "Gehe zu Schaubild"
+                      "label":"Kommentare",
```

Comment:
Will semantics.json also be changed? As of this position, quite some translations have changed and don't match with semantics. There, "Title" is followed by "Go To slide" which corresponds to the old translation ("Titel" and "Gehe zu Schaubild"). The same seems to be true for other translations.